### PR TITLE
Refactored 2.4GHz regulatory domain defines

### PR DESCRIPTION
**Pull request by FOG-Yamato**
Removed the `#define Regulatory_Domain_ISM_2400 1` and made it just a simple define while also moving it to the feature section of the targets.

### CODE REVIEW

File: src/lib/FHSS/FHSS.cpp [hunk 1]
```diff
@@ -132,7 +132,7 @@ const uint32_t FHSSfreqs[] = {
     FREQ_HZ_TO_REG_VAL(925700000),
     FREQ_HZ_TO_REG_VAL(926300000),
     FREQ_HZ_TO_REG_VAL(926900000)};
-#elif Regulatory_Domain_ISM_2400
+#elif defined Regulatory_Domain_ISM_2400
```
Comment (pkendall64) [inline src/lib/FHSS/FHSS.cpp:135]: parens, like the others

**Review by pkendall64 — APPROVED** [review]

File: src/lib/FHSS/FHSS.cpp (hunk 1 above)
Comment (FOG-Yamato) [reply src/lib/FHSS/FHSS.cpp:135]: FHSS file doesn't use any parentheses for the hopping frequencies so I left it without for the sake of consistencies

Comment (pkendall64) [reply src/lib/FHSS/FHSS.cpp:135]: Ok

**Comment by JyeSmith** [comment]
@cruwaller can you please review.  Any reason for/against this?

**Comment by cruwaller** [comment]
This is just a nice to have code formatting PR... Should have been discussed in #1001 before merge.

Group to decide which is the code format the ELRS uses now and in the future 😎

File: src/include/target/DIY_2400_TX_ESP32_SX1280_E28.h [hunk 2]
```diff
@@ -3,8 +3,7 @@
 #endif
 
 // Any device features
-#define HAS_OLED
-#define HAS_OLED_SPI
+#define USE_OLED_SPI
```
Comment (cruwaller) [inline src/include/target/DIY_2400_TX_ESP32_SX1280_E28.h:6]: These should be in separate PR. Not related to regulatory domain defines.

**Review by cruwaller — CHANGES_REQUESTED** [review]

**Comment by cruwaller** [comment]
Sorry bad result, github shown me merge changes :(
